docs/technical_specs/core/solvers.rst

```diff
@@ -74,17 +92,18 @@ Windows installation. The easiest option is to run the available precompiled
 Linux version via the WSL
 (see :ref:`binary installation <getting_started/binaries:Using the WSL>`
 for details). Expert users may wish to compile their own solver. Source code is
-available in the `idaes-ext repo <https://github.com/IDAES/idaes-ext/tree/main/petsc>`_
+available in the `idaes-ext repo <https://github.com/IDAES/idaes-ext/tree/main/petsc>`_.
 If you can compile PETSc for Windows, compiling the interface is trivial (see
 `PETSc's windows installation documentation <https://petsc.org/main/install/windows/>`_).
 
 The IDAES PETSc package also includes Python modules for reading binary data
 written by the PETSc solver.  On Windows, some manual installation of the Python
 modules is required.  If you are using the WSL method to run PETSc, copy the
 ``petscpy`` directory from the Linux package you are using to the IDAES binary
-directory.  You can find the IDAES binary directory with the command
-``idaes bin-directory``.  The primary use for these Python modules is to read
-trajectory files saved by the TS solver.
+directory.  You can find the IDAES binary directory by running the command
+``idaes bin-directory`` in the OS command shell (e.g. Bash, Windows CMD,
+PowerShell).  The primary use for these Python modules is to read trajectory
+files saved by the TS solver.
```

Comment:
May be worth mentioning that, if you're using something like Miniconda to manage multiple environments, it has to be an environment where you installed IDAES